adhocteam/Head-Start-TTADP
# Allow to filter grants/grantees based on region

**Description of change**
A user is only given a list of grantees/grants that they have permissions to.
Allowable permissions are defined as Admin, Read, Read+Write.

**How to test**
Pull changes and run
Go start a report and you should only see grants/grantee for section 14 (Assuming you are user 1 from seed data)

*Previous Behaviour*
When seeing the grant/grantee list you would see region 14 and region 9

**Issue(s)**
* https://github.com/HHS/Head-Start-TTADP/issues/154

**Checklist**

    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified
